tests/core/pyspec/eth2spec/test/phase0/epoch_processing/test_process_justification_and_finalization.py

```diff
@@ -300,3 +302,75 @@ def test_12_ok_support_messed_target(spec, state):
 @spec_state_test
 def test_12_poor_support(spec, state):
     yield from finalize_on_12(spec, state, 3, False, False)
+
+
+@with_all_phases
+@spec_state_test
+def test_balance_threshold_with_exited_validators(spec, state):
+    """
+    This test exercises a very specific failure mode where
+    exited validators are incorrectly included in the total active balance
+    when weighing justification.
+    """
+    rng = Random(133333)
+    # move past genesis conditions
+    for _ in range(3):
+        next_epoch_via_block(spec, state)
+
+    # mock attestation helper requires last slot of epoch
+    for _ in range(spec.SLOTS_PER_EPOCH - 1):
+        next_slot(spec, state)
+
+    # Step 1: Exit ~1/2 vals in current epoch
+    epoch = spec.get_current_epoch(state)
+    for index in spec.get_active_validator_indices(state, epoch):
+        if rng.choice([True, False]):
+            continue
+
+        validator = state.validators[index]
+        validator.exit_epoch = epoch
+        validator.withdrawable_epoch = epoch + 1
```

Comment:
`+ 1` is unrealistic. Suggest using the min withdrawability constant. Might save us trouble when writing withdrawal tests in the future